src/zenml/cli/stack_components.py

```diff
@@ -69,47 +76,93 @@ def generate_stack_component_get_command(
 ) -> Callable[[], None]:
     """Generates a `get` command for the specific stack component type."""
 
+    display_name = _component_display_name(component_type)
+
     def get_stack_component_command() -> None:
         """Prints the name of the active component."""
-        active_stack = Repository().active_stack
-        component = active_stack.components.get(component_type, None)
-        display_name = _component_display_name(component_type)
 
-        if component:
-            cli_utils.declare(f"Active {display_name}: {component.name}")
-        else:
-            cli_utils.warning(
-                f"No {display_name} set for active stack ({active_stack.name})."
-            )
+        cli_utils.print_active_profile()
+        cli_utils.print_active_stack()
+
+        repo = Repository()
+
+        local_stack = repo.active_stack_name
+        global_stack = repo.active_profile.active_stack or ""
+
+        stacks = {
+            "global": global_stack,
+            "local": local_stack,
+        }
+
+        for scope, stack_name in stacks.items():
+            if not stack_name:
+                continue
+            active_stack = repo.get_stack(stack_name)
+            component = active_stack.components.get(component_type, None)
+
+            if component:
+                cli_utils.declare(
+                    f"Active {scope} {display_name}: {component.name}"
+                )
+            else:
+                cli_utils.warning(
+                    f"No {display_name} set for {scope} active stack "
+                    f"({active_stack.name})."
+                )
 
     return get_stack_component_command
 
 
 def generate_stack_component_describe_command(
     component_type: StackComponentType,
-) -> Callable[[Optional[str]], None]:
+) -> Callable[[Optional[str], bool], None]:
     """Generates a `describe` command for the specific stack component type."""
 
+    singular_display_name = _component_display_name(component_type)
+
     @click.argument(
         "name",
         type=str,
         required=False,
     )
-    def describe_stack_component_command(name: Optional[str]) -> None:
+    @click.option(
+        "--global",
+        "-g",
+        "global_profile",
+        is_flag=True,
+        help=f"Show the global active {singular_display_name}.",
```

Comment:
"describe the stack component from global" maybe?

Reply:
I removed this entirely.